File: Web/godofgeeks23_AviralSrivastava_2024cse1056_2/gettingStarted/printallprimestilln.cpp
#include <iostream>
#include <cmath>
using namespace std;
int isPrime(int n)
{
    if(n==1)
        return 0;
    for(int i=2;i<=sqrt(n);i++)
        if(n%i==0)
            return 0;
    return 1;
}
int main(){
    int low, high;
    cin >> low >> high;
    for(int i=low;i<=high;i++)
        if(isPrime(i)==1)
            cout<<i<<endl;
}

File: AR-VR/thisiskartikgupta_KartikGupta_1923co1043_3/GettingStarted/PrintAllPrimesTillN.java
import java.util.*;

public class PrintAllPrimesTillN {

  static boolean isPrime(int n) {
    for(int i = 2; i * i <= n; i++)
        if(n % i == 0)
            return false;

    return true;
}

  public static void main(String[] args) {
      // write your code here

      Scanner sc = new Scanner(System.in);

      int low = sc.nextInt();
      int high = sc.nextInt();

      for(int i = low; i <= high; i++)
      if(isPrime(i))
          System.out.println(i);
  }
}